lib/components/Form.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Class representing a generic Form
+ *
+ * This component returns a <form> with legend.
+ *
+ * Required props:
+ * - legend: string. sets the form legend
+ * - children: node. Series of field control types
+ *
+ * Optional props:
+ * - id: string. Sets the form's id attribute
+ */
+
+export default class Form extends Component { 
+  render() {
+    return (
+      <div>
+        <form className="usa-form">
+          <fieldset className="usa-fieldset-inputs usa-sans">
```

Comment:
I'm not sure we always want a top-level fieldset.  Fieldsets should be used to group related form elements together so having a form-global one doesn't make a lot of sense.  Thoughts?